RawVideo/RawVideo/ContentView.swift

```diff
@@ -0,0 +1,1032 @@
+                        break
+                    case .screenShareOutgoing:
+                        StopScreenCaptureService()
+                        break
+                    default:
+                        break
+                }
+            
+                break
+            default:
+                break
+        }
+    }
+    
+    public func OnIncomingVideoStreamStateChanged(stream: IncomingVideoStream) -> Void
+    {
+        if (incomingVideoStream != nil && incomingVideoStream != stream)
+        {
+            if (stream.state == .available)
+            {
+                Task
+                {
+                    await ShowMessage(message: "This app only support 1 incoming video stream from 1 remote participant")
+                }
+            }
+            
+            return
+        }
+        
+        switch (stream.state)
+        {
+            case .available:
+                switch (stream.type)
+                {
+                case .remoteIncoming:
+                    remoteVideoStream = stream as? RemoteVideoStream
+                    StartRemotePreview()
+                    
+                    break
+                case .rawIncoming:
+                    rawIncomingVideoStream = stream as? RawIncomingVideoStream
+                    rawIncomingVideoStream!.delegate = rawIncomingVideoStreamObserver
+                    rawIncomingVideoStream!.start()
+                    
+                    break
+                default:
+                    break
+                }
+                
+                incomingVideoStream = stream;
+                break
+            case .stopped:
+                if (incomingVideoStreamType == .remoteIncoming)
+                {
+                    StopRemotePreview()
+                }
+            
+                break
+            case .notAvailable:
+                if (incomingVideoStreamType == .rawIncoming)
+                {
+                    rawIncomingVideoStream!.delegate = nil
+                }
+                
+                incomingVideoStream = nil
+                break
+            default:
+                break
+        }
+    }
+    
+    func OnRawVideoFrameArrived(rawVideoFrameBuffer: RawVideoFrameBuffer) -> Void
+    {
+        incomingPixelBuffer = rawVideoFrameBuffer.buffer
+    }
+    
+    func OnRawVideoFrameCaptured(rawVideoFrameBuffer: RawVideoFrameBuffer) -> Void
+    {
+        outgoingPixelBuffer = rawVideoFrameBuffer.buffer
+    }
+    
+    private func StartRemotePreview() -> Void
+    {
+        if (incomingVideoStreamRendererView == nil)
+        {
+            incomingVideoStreamRenderer = try! VideoStreamRenderer(remoteVideoStream: remoteVideoStream!)
+            let options = CreateViewOptions.init(scalingMode: ScalingMode.fit)
+            incomingVideoStreamRendererView = try! incomingVideoStreamRenderer?.createView(withOptions: options)
+        }
+    }
```

Comment:
Suggestion, let's avoid this `try!` as it would crash if error is thrown.
```suggestion
    private func StartRemotePreview() throws {
        if incomingVideoStreamRendererView == nil {
            incomingVideoStreamRenderer = try VideoStreamRenderer(remoteVideoStream: remoteVideoStream!)
            let options = CreateViewOptions.init(scalingMode: ScalingMode.fit)
            incomingVideoStreamRendererView = try incomingVideoStreamRenderer?.createView(withOptions: options)
        }
    }
```

Reply:
done